From 19ebc65d2c5854d9dea6f4a710ebfa695c80f030 Mon Sep 17 00:00:00 2001
From: Riku Viitanen <riku.viitanen@protonmail.com>
Date: Tue, 22 Aug 2023 20:07:22 +0300
Subject: [PATCH] mb/hp8300usdt: enable mSATA

Enables the mSATA slot on HP Compaq Elite 8300 USDT.
Tested, it works at the same speed as vendor FW (3Gb/s).

Signed-off-by: Riku Viitanen <riku.viitanen@protonmail.com>
---
 src/mainboard/hp/compaq_elite_8300_usdt/devicetree.cb | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/mainboard/hp/compaq_elite_8300_usdt/devicetree.cb b/src/mainboard/hp/compaq_elite_8300_usdt/devicetree.cb
index ba4ac6d7f2..f7f321fc83 100644
--- a/src/mainboard/hp/compaq_elite_8300_usdt/devicetree.cb
+++ b/src/mainboard/hp/compaq_elite_8300_usdt/devicetree.cb
@@ -9,9 +9,9 @@ chip northbridge/intel/sandybridge
 			register "pcie_hotplug_map" = "{ 0, 0, 0, 0, 0, 0, 0, 0 }"
 			register "pcie_port_coalesce" = "1"
 			register "sata_interface_speed_support" = "0x3"
-			register "sata_port_map" = "0x3"  # 0x1: 2.5" slot
+			register "sata_port_map" = "0x7"  # 0x1: 2.5" slot
 			                                  # 0x2: DVD
-			                                  # 0x?: mSATA
+			                                  # 0x4: mSATA
 			register "spi_lvscc" = "0x2005"
 			register "spi_uvscc" = "0x2005"
 			register "superspeed_capable_ports" = "0x0000000f"
-- 
2.41.0

